aries-site/src/pages/templates/content-layouts.mdx

```diff
@@ -32,39 +71,89 @@ Specifying layout content in this way will also ensure that it is interepreted a
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/applications.json",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/constants.js",
+    ]}
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  screenContainer={{ scale: 0.75 }}
+>
+  <ContainerDrivenLayout />
+</Example>
 
-See the table below for breakpoints defined by the HPE Design System. Keep in mind that these breakpoints are not meant to target every single existing device &mdash; we understand that technology is ever evolving. Instead, the ranges defined here help build a strong foundation to account for diverse use cases when creating responsive designs.
+### Content-Driven Layouts
+Content-driven layouts are driven from the inside-out, allowing the container to respond 
+and adapt to things like user entered content, like names. The layout is custom fit to 
+the content it holds.
 
-| T-shirt Size   | Range         |  Margin (pixels) |
-| -------------- | ----------    | -----------------:
-| **xsmall**    | 576 and below |      12
-| **small**     | 577 - 768     |      24
-| **medium**      | 769 - 1080    |      24
-| **large**     | 1081 - 1439   |      48
-| **xlarge**      | 1440 and above|      48
+The content-driven approach is useful for layouts such as modal dialogs or side-drawer 
+content within a layer, where the content should occupy the space it needs and not be 
+artificially constrained by its layout container.
 
-#### Page Margin
+The following example takes a content-driven approach, implementing a modal dialog 
+displaying content returned from a server with an unknown length.
 
-Page margin is the left and right space between the main content of a page and the screen edge.
-Based on the screen size, the page margin differs at each breakpoint.
+### Nested Grids
+When dealing with complex layouts, it is helpful to simplify them by thinking of them 
+as nested grids. Any column or grid area may be a grid itself. 
 
-<Example 
-  caption="A diagram showing the margin area on a large and small screen." 
-  plain 
-  pad="small"
-  previewWidth="large"
-  height={{ min: 'small' }}
+<Example
+  horizontalLayout
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/NestedGrid.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/templates/content-layouts/NestedGrid.js"
+  caption="Diagram illustrating how Grids may be nested inside one another."
+  width={{ min: "large" }}
 >
-  <PageMarginAnatomy />
+  <NestedGrid />
```

Comment:
Would it be helpful to label the sub grid areas?

Such as A1, A2, B1, B2

OR use different colors for the nested grids?

Reply:
with so many nested dashed boxes, it's a little hard to understand where the nesting is happening on first glance.

Reply:
I was also thinking maybe different colors for this

Reply:
I tend to agree with being a bit confused by the number of boxes and also what the yellow color means, why some boxes have it and some don't. I'm wondering if we could simplify it by removing the dotted box surrounding nested grids A and B and also removing the dotted box around the two right yellow boxes within nested grid B.

Another way to think about this could be to consider a dotted line container to be a Grid and a yellow background container to be a Box, and then having nested grid A and B as direct children of parent grid. Just a though.

Reply:
done